dev/com.ibm.ws.cdi.beansxml_fat/fat/src/com/ibm/ws/cdi12/fat/tests/BeanDiscoveryModeNoneTest.java

```diff
@@ -56,19 +63,25 @@ public static Archive buildShrinkWrap() {
      * @see com.ibm.ws.fat.LoggingTest#getSharedServer()
      */
     @Override
-    protected ShrinkWrapSharedServer getSharedServer() {
-        return SHARED_SERVER;
+    protected SharedServer getSharedServer() {
+        return null;
     }
 
     @Test
     @ExpectedFFDC({ "javax.servlet.UnavailableException", "com.ibm.wsspi.injectionengine.InjectionException" })
     public void testcorrectExceptionThrown() throws Exception {
-        this.verifyStatusCode("/beanDiscoveryModeNone/TestServlet", 404);
+        WebBrowser webBrowser = WebBrowserFactory.getInstance().createWebBrowser((File) null);
+        int statusCode = 404;
+        int actualStatusCode = webBrowser.request(createURL("/beanDiscoveryModeNone/TestServlet"), statusCode);
+
+        if (actualStatusCode != statusCode) {
+            throw new WebBrowserException("The expected status code was not thrown.  Expected " + statusCode + ", was " + actualStatusCode);
+        }
```

Comment:
We have a class `HttpRequest` which can create the URL for you and is a little simpler.

I think you'd do
```
new HttpRequest(server, "/beanDiscoveryModeNone/TestServlet").expectCode(404).run(String.class);
```